### PR TITLE
Swtiched intake to use one talon

### PR DESCRIPTION
**Pull request by VGopal06**

### CODE REVIEW

File: src/main/java/frc/robot/constants/IntakeConstants.java [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
 
-import edu.wpi.first.math.controller.SimpleMotorFeedforward;
 import edu.wpi.first.math.util.Units;
 
 public final class IntakeConstants {
-    public static final String LEFT_SPARK = "intake spark 1";
+    public static final String TALON = "intake spark 1";
     public static final int LEFT_SPARK_ID = 1;
     public static final String RIGHT_SPARK = "intake spark 2";
     public static final int RIGHT_SPARK_ID = 2;
```
Comment (Amehta996) [inline src/main/java/frc/robot/constants/IntakeConstants.java:9]: 1. Rename it from intake spark 1 to just intake falcon
2. Remove the Right Spark
3. Change the `Left_Spark_ID` to Intake Falcon ID and set it to 1
4. Remove `RIGHT_SPARK_ID.

File: src/main/java/frc/robot/di/devices/MotorsModule.java [hunk 2]
```diff
@@ -88,41 +88,13 @@ public WPI_TalonFX elevatorMotor() {
...
-        sparkMax.setPeriodicFramePeriod(CANSparkMaxLowLevel.PeriodicFrame.kStatus1, SparkMaxConstants.MEDIUM_PRIORITY_MS);
-        sparkMax.setPeriodicFramePeriod(CANSparkMaxLowLevel.PeriodicFrame.kStatus2, SparkMaxConstants.MEDIUM_PRIORITY_MS);
-        sparkMax.setPeriodicFramePeriod(CANSparkMaxLowLevel.PeriodicFrame.kStatus3, SparkMaxConstants.LOW_PRIORITY_MS);
-        sparkMax.setPeriodicFramePeriod(CANSparkMaxLowLevel.PeriodicFrame.kStatus4, SparkMaxConstants.LOW_PRIORITY_MS);
-
-        return sparkMax;
+    @Named(IntakeConstants.TALON)
+    public WPI_TalonFX providesIntakeTalon() {
+        WPI_TalonFX talon = new WPI_TalonFX(IntakeConstants.LEFT_SPARK_ID);
+        talon.setInverted(false);
```
Comment (Amehta996) [inline src/main/java/frc/robot/di/devices/MotorsModule.java:94]: 1. Add CanBus frame timings

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 3]
```diff
@@ -1,46 +1,44 @@
...
-import org.opencv.core.Mat;
 
 import javax.inject.Inject;
 import javax.inject.Named;
 
 public class IntakeSubsystem extends SubsystemBase {
-    private final CANSparkMax leftSpark;
-    private final CANSparkMax rightSpark;
+
+    private final WPI_TalonFX talon;
```
Comment (Amehta996) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java:15]: refactor talon to intakeMotor

**Review by Amehta996 — CHANGES_REQUESTED** [review]
Look at comments made

**Review by vwu1888 — APPROVED** [review]
lgtm

**Pull request merged**